From 77099c50d7afd43fb9f4699085406fbfed49d006 Mon Sep 17 00:00:00 2001
From: Joe Slater <jslater@windriver.com>
Date: Thu, 31 Jan 2019 11:09:22 +0800
Subject: [PATCH] refpolicy: modify selinuxutil.te

We need to be able to create and delete links under
/etc/selinux/<policy>.  manage_lnk_file_perms is
probably more than we need, but it is a safe choice.

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>
---
 policy/modules/system/selinuxutil.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/system/selinuxutil.te b/policy/modules/system/selinuxutil.te
index 1452ffd..414a7f1 100644
--- a/policy/modules/system/selinuxutil.te
+++ b/policy/modules/system/selinuxutil.te
@@ -487,6 +487,8 @@ filetrans_pattern(semanage_t, selinux_config_t, semanage_store_t, dir, "modules"
 
 allow semanage_t semanage_tmp_t:dir manage_dir_perms;
 allow semanage_t semanage_tmp_t:file { manage_file_perms mmap_exec_file_perms };
+allow semanage_t selinux_config_t:lnk_file manage_lnk_file_perms;
+
 files_tmp_filetrans(semanage_t, semanage_tmp_t, { file dir })
 
 kernel_read_system_state(semanage_t)
-- 
2.7.4

